pyproject.toml

```diff
@@ -43,6 +43,7 @@ dev = [
     "hatch",
     "matplotlib",
     "jupyter==1.0.0",
+    "types-pytz",
```

Comment:
What is this package for? I know `pytz` is already part of the Python standard library.

Reply:
It's the typing information for `pytz`, needed by mypy